<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1677</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Input_Image">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>521</width>
      <height>531</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Image">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>720</y>
      <width>341</width>
      <height>27</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="pushButton_load_input_image">
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_compute_on_original">
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Compute on original image</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_edit_mode">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>60</y>
      <width>271</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_gradient_experiment_config">
     <attribute name="title">
      <string>Gradient Experiment</string>
     </attribute>
     <widget class="QSpinBox" name="spinBox_alternate_every_n">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Alternate every</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>221</width>
        <height>431</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Merge Mode</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_merge_mode"/>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Multiplier Mode</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_multiplier_mode"/>
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSpinBox" name="spinBox_multiplier_amplitude">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Multiplier Amplitude</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSpinBox" name="spinBox_multiplier_frequency">
        <property name="value">
         <number>16</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Multiplier Frequency</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QRadioButton" name="radioButton_clip_images">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Clip Images</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_reset_output">
       <property name="text">
        <string>Reset Output</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Output start image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_output_start_image"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_schnock_original_config">
     <attribute name="title">
      <string>Schnock Original</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Mid</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>High</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Shift </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSpinBox" name="spinBox_high_shift"/>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Threshold</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLCDNumber" name="lcdNumber_low_threshold"/>
       </item>
       <item row="4" column="2">
        <widget class="QLCDNumber" name="lcdNumber_mid_threshold"/>
       </item>
       <item row="4" column="3">
        <widget class="QLCDNumber" name="lcdNumber_high_threshold"/>
       </item>
       <item row="5" column="2">
        <widget class="QSlider" name="verticalSlider_mid_threshold">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>150</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QSlider" name="verticalSlider_high_threshold">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>220</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Low</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spinBox_mid_shift"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QSpinBox" name="spinBox_low_shift"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QSlider" name="verticalSlider_low_threshold">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_foto_or_video">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>700</y>
      <width>481</width>
      <height>121</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_photo_mode">
     <attribute name="title">
      <string>Photo Mode</string>
     </attribute>
     <widget class="QSplitter" name="splitter_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>441</width>
        <height>24</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="pushButton_process_image">
       <property name="text">
        <string>Process Image</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_continuous">
       <property name="text">
        <string>Continuous</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_fast_forward_iterations">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Fast forward iterations</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_process_folder">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>108</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Process Folder</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_video_mode">
     <attribute name="title">
      <string>Video Mode</string>
     </attribute>
     <widget class="QSplitter" name="splitter_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QRadioButton" name="radioButton_webcam_input">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Webcam input</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Video merge mode:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_video_merge_mode"/>
     </widget>
     <widget class="QSplitter" name="splitter_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>381</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Webcam delay:</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_webcam_delay"/>
      <widget class="QSlider" name="horizontalSlider_webcam_delay">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_save_image">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>630</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1677</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionLoad_Image"/>
    <addaction name="actionExport_Image"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoasd">
   <property name="text">
    <string>Load project</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionSave_configurations">
   <property name="text">
    <string>Save configurations</string>
   </property>
  </action>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="actionExport_Image">
   <property name="text">
    <string>Export Image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
